YACReader/magnifying_glass.cpp

```diff
@@ -141,31 +141,34 @@ void MagnifyingGlass::updateImage()
 }
 void MagnifyingGlass::wheelEvent(QWheelEvent *event)
 {
+    // TODO: consider handling horizontal scrolling differently.
+    const auto delta = event->angleDelta().x() + event->angleDelta().y();
+    const bool grow = delta < 0;
```

Comment:
The zoom in/out direction of the Magnifying glass is opposite to the direction in other applications. Is this intentional or an oversight that should be fixed by initializing `grow` with `delta > 0`?